### ISSUE TITLE
Articles editor Content tab stuck on loading

### ISSUE THREAD

**Issue by joeacarstairs** [joomla/joomla-cms#39565]
### Steps to reproduce the issue

Go to Articles. Select any existing Article to open the editor. Ensure the Content tab is selected.

### Expected result

I should be able to edit the article content.

### Actual result

I get stuck with an eternal loader (see screenshot below).

![screen shot 2023-01-06 at 18 28 54](https://issues.joomla.org/uploads/1/f85c209ea42d958526a073b4d34a0ab2.png)

### System information (as much as possible)

PHP version: 8.0.16
Joomla! version: 4.2.6 Stable [ Uaminifu ]
Browser: Mozilla Firefox 109.0b8, Google Chrome 108.0.5359.125
MySQL version: 10.3.34-MariaDB 
Web server: Apache

**Comment by wojtekxtx** [comment]
@Sycamost try clearing browser's cache.

**Comment by brianteeman** [comment]
Most likely there will be a jacascript error shown in the browser console showing you the problem.

From the screenshot I can see that you are either using a third party editor or a third party content plugin. This is almost certainly the cause of the problem which the javascript error will confirm

**Comment by Sycamost** [comment]
@wojtekxtx Thanks for the suggestion, but clearing the cache had no effect.

@brianteeman Spot on! I looked in the browser console and it looks like ArkEditor is at fault. I switched the default editor to JCEEditor and we're back in business! I'll let Ark know about the issue, but in the meantime I can edit in peace.

![image](https://user-images.githubusercontent.com/65492573/211141210-45e85ba1-c3be-4560-912d-0cf0f46637a5.png)
